packages/lab-extension/src/index.ts

```diff
@@ -90,22 +91,33 @@ const interfaceSwitcher: JupyterFrontEndPlugin<void> = {
     };
 
     const addInterface = (option: ISwitcherChoice) => {
-      const { command, commandLabel, urlPrefix } = option;
+      const { command, commandLabel, commandDescription, urlPrefix } = option;
+      const descriptionCommand = `${command}-description`;
+
+      const execute = () => {
+        const current = notebookTracker.currentWidget;
+        if (!current) {
+          return;
+        }
+        window.open(`${urlPrefix}${current.context.path}`);
+      };
+
       commands.addCommand(command, {
         label: (args) => (args.noLabel ? '' : commandLabel),
         caption: commandLabel,
-        execute: () => {
-          const current = notebookTracker.currentWidget;
-          if (!current) {
-            return;
-          }
-          window.open(`${urlPrefix}${current.context.path}`);
-        },
+        execute,
+        isEnabled,
+      });
+
+      commands.addCommand(descriptionCommand, {
```

Comment:
Wondering if there is a way to use the `isPalette` trick mentioned in https://github.com/jupyter/notebook/pull/6809#issuecomment-1492958716.

To avoid adding "proxy" commands to the commands registry.

Reply:
Done in https://github.com/jupyter/notebook/pull/6847/commits/52d12b593809bdcde13edeff5163b26c978d3f9e

Reply:
So that's where you provide args! Thank you @jtpio